### DISCUSSION TITLE
how can i start contributing the Giskard??

### DISCUSSION THREAD

**Discussion by Atif1727** [Giskard-AI/giskard-oss#1742] [General]
I am new to Giskard i have seen the github page but unable to understant how can i setup the dev enivronment.i am not getting any quickstart or contribution guideliness for setup project.

**Comment by luca-martial** [comment]
Hey @Atif1727 ! Thanks for asking to contribute. We have a [section in our docs](https://docs.giskard.ai/en/latest/community/contribution_guidelines/index.html) regarding this. I would encourage you to find GitHub issues with the `good first issue` tag. Our contributors generally start there. Let me know if you run into any challenges!
